lisa/wa.py

```diff
@@ -552,3 +553,61 @@ def __init__(self, wa_output, trace_to_df, **kwargs):
     def _get_artifact_df(self, path):
         trace = Trace(path, **self._trace_kwargs)
         return self._trace_to_df(trace)
+
+class WAJankbenchCollector(WAArtifactCollectorBase):
+    """
+    WA collector for the jankbench frame timings.
+
+    The collector framework will return a single :class:`pandas.DataFrame` with the
+    results from every jankbench job in WACollector database format (i.e. the returned dataframe
```

Comment:
nit: s/WACollector/:class`lisa.stats.Stats`/ as the format is described by that class.